node/test/dialog.spec.ts

```diff
@@ -135,16 +135,21 @@ describe('dialog', async () => {
     it(`gives expected responses to scenario inputs: ${ex.name}`, async () => {
       if (mockAxios) {
         mockAxios.reset();
+        mockAxios.onGet('/config').reply(() => {
```

Comment:
we shouldn't need this and instead use `beforeEach` to set `process.env.API_KEY` (and even then only if you're spying/testing that it gets sent to the mock graphql service?)